ovh/resource_ovh_ip_reverse.go

```diff
@@ -0,0 +1,154 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type OvhIpReverse struct {
+	IpReverse string `json:"ipReverse"`
+	Reverse   string `json:"reverse"`
+}
+
+func resourceOvhIpReverse() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOvhIpReverseCreate,
+		Read:   resourceOvhIpReverseRead,
+		Update: resourceOvhIpReverseUpdate,
+		Delete: resourceOvhIpReverseDelete,
+
+		Schema: map[string]*schema.Schema{
+			"ip": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"ipreverse": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					err := validateIp(v.(string))
+					if err != nil {
+						errors = append(errors, err)
+					}
+					return
+				},
+			},
+			"reverse": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func resourceOvhIpReverseCreate(d *schema.ResourceData, meta interface{}) error {
+	provider := meta.(*Config)
+
+	// Create the new reverse
+	newReverse := &OvhIpReverse{
+		IpReverse: d.Get("ipreverse").(string),
+		Reverse:   d.Get("reverse").(string),
+	}
+
+	newIp, ok := d.GetOk("ip")
+	if !ok {
+		newIp = newReverse.IpReverse
+		d.Set("ip", newIp)
+	}
+
+	log.Printf("[DEBUG] OVH IP Reverse create configuration: %#v", newReverse)
+
+	resultReverse := OvhIpReverse{}
+
+	var err error
+
+	for i := 0; i < 5; i++ {
```

Comment:
why?
i think we should leave this "retry management" to the end user. if it fails, the end user can retry to apply the plan.

Reply:
Since ovh terraform provider already support dns resources, we should wait the apply of dns before be able to set the reverse dns. He handles retries in order to wait for dns propagation.
Am I right @pkuhner?

Reply:
well that's not the way you should design your plan.
dns propagation is not garantueed, even if the task status is done. it could last several hours